controllers/comments.js

```diff
@@ -22,4 +22,13 @@ exports.postReviewComment = (req, res, next) => {
         .catch((err) => {
             next(err)
     });
-};
\ No newline at end of file
+};
+
+exports.deleteComment = (req, res, next) => {
+    const { comment_id } = req.params;
+    removeComment(comment_id).then(() => {
+        res.status(204).send();
+    }).catch((err) => {
```

Comment:
Hey Karly! As we are not sending any body back with our response I encourage you to research [this](http://expressjs.com/en/5x/api.html#res.sendStatus) to see how we can just send back the status code!